CONTRIBUTING.md

```diff
@@ -1,13 +1,37 @@
 ## How to Contribute to Scene Builder
 
-Contribution of any form is welcome! Please see the list below on how you can contribute to the project. Once you've decided what you would like to do, let us know about it first in the [Discussions section](https://github.com/gluonhq/scenebuilder/discussions). This is to make sure that the issue you want hasn't already been implemented or being worked on in newer versions. Any new API or changes to existing API should be discussed to avoid inconsistencies.
+Contribution of any form is welcome! Please see the list below on how you can contribute to the project. 
 
 * Provide or suggest an implementation of an issue from [GitHub Issues](https://github.com/gluonhq/scenebuilder/issues).
 * Proof read the [public documentation](https://github.com/gluonhq/scenebuilder/wiki) for errors, ambiguities and typos.
 * Create an issue or suggest a feature backed up by a use case.
 * Add missing tests.
+* Quality-test early release builds.
+
+## Contribution Workflow
+
+1. Pick an unassigned [open issue](https://github.com/gluonhq/scenebuilder/issues).
+2. Comment on it to say you would like to contribute to fixing it and propose a plan (if appropriate).
+3. A maintainer will confirm the issue is valid and can be assigned to you for a fix.
+4. You produce a Pull Request following the Standards below.
+5. Once you tick all check boxes on the Pull Request template, it will be reviewed by a maintainer or a community member.
+6. Once the Pull Request has at least one approval, it will be ready for a merge.
 
 ## Coding Standards
 
 * Any code contribution should follow [the OpenJFX guidelines](https://github.com/openjdk/jfx/blob/master/CONTRIBUTING.md#coding-style-and-testing-guidelines).
-* Any Pull Request should follow the provided template.
\ No newline at end of file
+* Any Pull Request should follow the provided template.
+
+## Pull Request Standards
+
+The project uses the following Pull Request message guidelines, based on [Conventional Commits 1.0.0](https://www.conventionalcommits.org/en/v1.0.0/):
+
+* **build: message** -- change affects the build system, configuration files, scripts, or external dependencies
+* **docs: message** -- change affects documentation **only**, including LICENSE, CONTRIBUTING, README
+* **feat: message** -- change adds a new or modifies an existing feature
+* **fix: message** -- change fixes a bug
+* **perf: message** -- change is related to performance
+* **refactor: message** -- change cleans up or restructures code, including formatting only changes
+* **test: message** -- change that adds new or updates existing tests and mostly affects the `test` package
```

Comment:
We may want to add a configuration file for the [semantic pull request bot](https://github.com/zeke/semantic-pull-requests#configuration) as a part of this PR?